lib/daru/io/io.rb

```diff
@@ -124,20 +124,24 @@ def dataframe_write_csv dataframe, path, opts={}
 
       def from_sql dbh, query
         require 'dbi'
+
         sth     = dbh.execute(query)
         vectors = {}
-        fields  = []
-        sth.column_info.each do |c|
-          vectors[c[:name]] = Daru::Vector.new([])
-          vectors[c[:name]].rename c[:name]
-          fields.push(c[:name].to_sym)
+        fields = []
+        sth.column_names.each do |column_name|
+          name = column_name.to_sym
+          vectors[name] = Daru::Vector.new([])
+          vectors[name].rename name
+          fields.push name
         end
-        ds=Daru::DataFrame.new(vectors,order: fields)
+
+        df = Daru::DataFrame.new(vectors, order: fields)
+
         sth.fetch do |row|
-          ds.add_row(row.to_a)
+          df.add_row(row.to_a)
         end
-        ds.update
-        ds
+
+        df
```

Comment:
`df.update` will ensure that missing values are duly updated in the resulting dataframe. Removing it might lead to wrong results.

You can keep an option in the `from_sql` method, something like `:update_missing_values` and set the default to `true`. This can serve as a switch for updating the track of missing values that Daru::DataFrame keeps. This way people who dont care about the missing values being updated can disable the update from running.

An `if opts[:update_missing_values]` statement around the `df.update` will serve as the switch.

Also, don't forget to add documentation describing the new option. We follow YARD for docs. You can refer to other methods to see how it can be done.